mapper/orm/src/main/java/org/hibernate/search/mapper/orm/cfg/HibernateOrmAutomaticIndexingSynchronizationStrategyName.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.orm.cfg;
+
+import java.lang.invoke.MethodHandles;
+
+import org.hibernate.search.mapper.orm.logging.impl.Log;
+import org.hibernate.search.util.common.impl.StringHelper;
+import org.hibernate.search.util.common.logging.impl.LoggerFactory;
+
+public enum HibernateOrmAutomaticIndexingSynchronizationStrategyName {
```

Comment:
We use it for every such enum. Mainly to avoid naming conflicts if we want to one day expose an interface to be implemented by the user.
Here it is not strictly necessary, but I'd rather stay consistent.